packages/shared/src/hooks/usePostModalNavigation.ts

```diff
@@ -221,6 +221,11 @@ export const usePostModalNavigation = (
         }
 
         const current = items[openedPostIndex] as PostItem;
+
+        if (!current) {
+          return;
+        }
```

Comment:
This would just throw error in the background when you press `left | right` arrow keys when article modal is not open. With this we only toggle logic when modal is opened.